Tests/scripts/hook_validations/integration.py

```diff
@@ -152,6 +153,27 @@ def is_default_arguments(self):
                                     .format(arg_name, command_name))
         return self._is_valid
 
+    def is_isarray_arguments(self):
+        """Check if a reputation command's (domain/email/file/ip/url)
+            argument of the same name has the 'isArray' attribute set to True
+
+        Returns:
+            bool. Whether 'isArray' is True
+        """
+        reputation_commands = ['file', 'email', 'domain', 'url', 'ip']
```

Comment:
lets make this a const

Reply:
I'm going to make this a class variable. cool?

Reply:
switched to constant